src/Evolution/Systems/CurvedScalarWave/Initialize.hpp

```diff
@@ -0,0 +1,90 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Tags.hpp"
+#include "Evolution/Initialization/InitialData.hpp"
+#include "Evolution/Initialization/Tags.hpp"
+#include "Evolution/Systems/CurvedScalarWave/System.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
+#include "PointwiseFunctions/AnalyticData/Tags.hpp"
+#include "Time/Time.hpp"
+
+namespace CurvedScalarWave::Initialization {
+/// \ingroup InitializationGroup
+/// \brief Initialize items adding constraint damping parameters of the
+/// CurvedScalarWave system
+///
+/// DataBox changes:
+/// - Adds:
+///   * `CurvedScalarWave::Tags::ConstraintGamma1`
+///   * `CurvedScalarWave::Tags::ConstraintGamma2`
+/// - Removes: nothing
+/// - Modifies: nothing
+///
+/// \note This action relies on the `SetupDataBox` aggregated initialization
+/// mechanism, so `Actions::SetupDataBox` must be present in the
+/// `Initialization` phase action list prior to this action.
```

Comment:
This is not an action, so delete this note and point to `Initialization::Actions::AddSimpleTags` instead (same for the other one below)